docs/Other SDL Features/Uploading Images/index.md

```diff
@@ -1,7 +1,9 @@
 # Uploading Images
 
 !!! NOTE
-If you use the @![iOS]`SDLScreenManager`!@@![android, javaSE, javaEE]`ScreenManager`!@, [image uploading for template graphics](Displaying a User Interface/Template Images), [soft buttons](Displaying a User Interface/Template Custom Buttons), and [menu items](Displaying a User Interface/Main Menu) is handled for you behind the scenes. However, you will still need to manually upload your images if you need images in an alert, VR help lists, turn-by-turn directions, or other features not currently covered by the @![iOS]`SDLScreenManager`!@ @![android, javaSE, javaEE]`ScreenManager`!@.
+If you use the @![iOS]`SDLScreenManager`!@@![android, javaSE, javaEE, javascript]`ScreenManager`!@, [image uploading for template graphics](Displaying a User Interface/Template Images), [soft buttons](Displaying a User Interface/Template Custom Buttons), and [menu items](Displaying a User Interface/Main Menu) is handled for you behind the scenes. However, you will still need to manually upload your images if you need images in an alert, VR help lists, turn-by-turn directions, or other features not currently covered by the @![iOS]`SDLScreenManager`!@ @![android, javaSE, javaEE, javascript]`ScreenManager`!@.
+
+The JavaScript manager does not currently handle uploading graphics for menu items. This will be included in a future version.
```

Comment:
```suggestion
@![javascript]
The JavaScript manager does not currently handle uploading graphics for menu items. This will be included in a future version.
!@
```